docs/release-instruction.md

```diff
@@ -34,12 +34,12 @@ Use `vX.Y.Z` tag following the semantic versioning standard.
 ## Push changes and release tag to Github
 
 This step should bypass the default PR mechanism to get a correct result (so
-that releasing requires admin privileges for the project), both the `master`
-branch update and tag must be pushed simultaneously like this:
+that releasing requires admin privileges for the project). Both the `master`
+branch update and the tag must be pushed simultaneously like this:
 
     $ git push origin master v0.70.1
 
-This is important to ensure proper version in a CI-built binary. Wait for
+It is important to ensure proper version in a CI-built binary. Wait for
```

Comment:
I think `this` is more appropriate here, we're talking about the magic push above.